main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+        for player_name, player_info in data.items():
+            race = player_info.get("race")
+            guild = player_info.get("guild")
+            skills = race.get("skills")
+
+            if race:
+                race, _ = Race.objects.get_or_create(
+                    name=race["name"],
+                    description=race["description"]
+                )
+
+            if guild:
+                guild, _ = Guild.objects.get_or_create(
+                    name=guild["name"],
+                    description=guild["description"]
+                )
+            if skills:
+                for skill in skills:
+                    Skill.objects.get_or_create(
+                        name=skill["name"],
+                        bonus=skill["bonus"],
+                        race=race
+                    )
+            if not None:
```

Comment:
I don’t get this? What do you try to check here

Reply:
"Race can be None due to .get method use player_info["race"] or .get but with default argument as dict
The same with skill and player"
Worked with Arsen's suggestions.